CIP-ShawnMcMurdo-NonCentralizingRankings/CIP-ShawnMcMurdo-NonCentralizingRankings.md

```diff
@@ -0,0 +1,100 @@
+---
+CIP: ?
+Title: Non-Centralizing Rankings
+Authors: Shawn McMurdo <shawn_mcmurdo@yahoo.com>
+Comments-URI: https://forum.cardano.org/t/how-to-improve-daedalus-rankings/40478
+Status: Draft
+Type: Standards
+Created: 2020-09-15
+License: CC-BY-4.0
+---
+
+## Summary
+
+Make Daedalus rankings more fair and non-centralizing by modifying the ranking methodology.
+
+## Abstract
+
+Modify the current ranking system by removing the centralizing Nash Equilibrium goal of the ranking methodology in order to give more fair rankings and improve the viability of the stake pool operator community and the network overall.  To do this we need to remove the stated goal of having k fully saturated pools and all other pools having no stake other than owner pledge, which goes against the Cardano goal of decentralization.
```

Comment:
It is true that the current ranking has a steep edge after the top k desirable pools. It seems totally reasonable to me to want to ease that transition. It's not clear to me, though, why we have to remove the equilibrium goal. It's also not obvious to me that this is more fair or more decentralized, it's just that there is less noise around the k boundary.

Reply:
@kevinhammond thanks for providing updates when feasible!